pcdc_aws_client/boto.py

```diff
@@ -450,6 +450,23 @@ def generate_presigned_url_for_uploading_part(bucket, key, uploadId, partNumber,
                     partNumber, key, e
                 )
             )
+    def get_squid_public_ips(self):
```

Comment:
Make this general so it can be used for other things beside squid. Something like get_ec2_public_ips_by_....